src/documentation/0009-node-environment-variables/index.md

```diff
@@ -10,6 +10,14 @@ The `process` core module of Node.js provides the `env` property which hosts all
 
 Here is an example that accesses the NODE_ENV environment variable, which is set to `development` by default.
 
+The below code runs `app.js` and set `USER_ID` and `USER_KEY`
```

Comment:
I think this part should be above the `Here's an example...` line

Reply:
Thanks, I updated it.